### ISSUE TITLE
Specification: Enable FREME NER to only spot entity without linking

### ISSUE THREAD

**Issue by jnehring** [freme-project/technical-discussion#48]
### Motivation

Internally FREME NER does at first entity spotting and then entity linking. Leaving out the linking part is expected to result in a huge performance gain and might be enough for certain applications.

We could introduce this feature along with measuring speed of FREME NER so we can easily measure the performance gain while we are measuring the rest anyways.

The goal of this issue is to specify the feature and find out whether it is feasible to do it within FREME 0.4.

### Work packages

* Create NIF model to encode such information.
* Give FREME NER the ability to produce this result.
* Enable this on the broker level. We could a) introduce a parameter `mode=spotting` and `mode=spotting+linking` to endpoint `/e-entity/freme-ner/documents`. Or b) we create a new endpoint. I propose b) because it is easier for the user when each endpoint has only one functionality. But it is also a quiet complex setup.
* Also standard work packages: Documentation, Testing, Bugfixing.

@nilesh-c do you have any comments on this? Any more work packages? Could you comment and find out which work package are difficult and how many days you expect to work on this?

**Comment by m1ci** [comment]
Very useful feature. Indeed we should implement this. Some comments bellow.

> Internally FREME NER does at first entity spotting and then entity linking. 

It does also `classification`. Read my comment bellow.

> Create NIF model to encode such information.

No need to introduce new NIF model. If one selects only "spotting" then `itsrdf:taIdentRef` triple will not be present in the output.

> Enable this on the broker level. We could a) introduce a parameter mode=spotting and 

I like the param name `mode=spotting`

> mode=spotting+linking to endpoint /e-entity/freme-ner/documents.

IMO it is better if split the modes with comma sign. E.g., `mode=spotting,linking`
Note that there is also `classification` step. Currently,  `spotting` is performed with StanfordNER and while performing `spotting` it does also `classification`. So, `classification` is inseparable from the spotting, and it is done in parallel. So when one set `mode=spotting` we can perform as usual `spotting` and `classification` and then filter out classes. Or, we just leave this info. What do you think. 

> Or b) we create a new endpoint. I propose b) because it is easier for the user when each endpoint has only one functionality. But it is also a quiet complex setup.

I would vote for a) implementing this as `mode` parameter of FREME NER - no need to maintain one more endpoint.

> @nilesh-c do you have any comments on this? Any more work packages? Could you comment and find out which work package are difficult and how many days you expect to work on this?

This feat should be easy to implement.
@nilesh-c any estimates on the time required?

**Comment by jnehring** [comment]
Moved to a new task: https://github.com/freme-project/freme-ner/issues/23
